src/glob-tool/templates/app.vue

```diff
@@ -137,13 +145,19 @@ limitations under the License.
                 shiftActive: false,
                 commentsEnabled: null,
                 matchesOnly: null,
+                matchOptions: {
+                    dot: false
+                },
                 hits: 0,
                 misses: 0,
             }
         },
         watch: {
             commentsEnabled() {
                 this.test()
+            },
+            "matchOptions.dot": function() {
+                this.test()
             }
```

Comment:
```suggestion
            matchOptions: {
                handler() {
                    this.test()
                },
                deep: true,
            },
```
To future-proof this if other settings are added, can we use a deep watcher for the whole object?